test/userRegistry.js

```diff
@@ -72,5 +81,12 @@ contract('UserRegistry', function (accounts) {
     assert.equal(tx, false)
     tx = await userRegistry.userDislikesVideo(userAddress, videoId)
     assert.equal(tx, true)
+
+//    likes_percentage, views, likes, dislikes = await videoRegistry.getStats(videoId)
+//    assert.equal(likes_percentage, 50)
+//    assert.equal(views, 0)
+//    assert.equal(likes, 1)
+//    assert.equal(dislikes, 1)
```

Comment:
This assert is wrong, I think. What happend is that `userAddress` first liked the video, then disliked it. the `dislike` should  cancel the like.